uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Helper/Menu/CiviHr.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+use CRM_HRCore_Menu_Config as MenuConfig;
+use CRM_HRCore_Menu_Item as MenuItem;
+
+
+class CRM_HRCore_Helper_Menu_CiviHr {
```

Comment:
I think that "helper" is a bit too generic here and might end up passing the wrong impression that we can add anything menu-related to this class. The first two ideas that come to my mind are `MenuBuilder` or `MenuFactory`. I slightly prefer the former, as `MenuFactory` could confuse people thinking that this would be an implementation of the Factory pattern.

Just to be clear, I mean that I think that the class name should be something like `CRM_HRCore_Menu_MenuBuilder` or  `CRM_HRCore_Menu_MenuFactory`